libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -449,6 +449,28 @@ AP_GPS::update(void)
 }
 
 /*
+  GPS time of week in days, hours, minutes, seconds, milliseconds
+*/
+void AP_GPS::time_week_utc(uint8_t instance, uint16_t &day, uint16_t &hour, uint16_t &min, uint16_t &sec, uint16_t &ms) const
```

Comment:
Wouldn't this be better to live in AP_GPS_Backend where other utils already are?